prompts/index.js

```diff
@@ -185,18 +189,17 @@ const noteContentsPrompt = [
   {
     type: 'input',
     name: 'content',
-    message: `Please enter your note ${reset.green('→')}`,
+    message: `Please enter your note`,
   },
 ]
 
 const snoozeDurationPrompt = [
   {
-    type: 'datepicker',
+    type: 'datetime',
     name: 'snoozeDuration',
-    message: `Until when would you like to snooze this incident? ${reset.green(
-      '→'
-    )}`,
+    message: `Until when would you like to snooze this incident`,
```

Comment:
I don't like this sentence, maybe change it to something like:

How long would you like to snooze this incident